packages/cms/store/auth.ts

```diff
@@ -31,9 +34,15 @@ export interface RegisterCredentials {
   password: string
 }
 
+export interface UpdateProfileCredentials {
```

Comment:
Rethinking this, maybe instead of credentials we should be calling them `Input`, for example `UpdateUserProfileInput`

Reply:
okay

Reply:
okay